include/nuttx/spinlock.h

```diff
@@ -351,6 +332,25 @@ void spin_clrbit(FAR volatile cpu_set_t *set, unsigned int cpu,
 
 #endif /* CONFIG_SPINLOCK */
 
+/****************************************************************************
+ * Name: spin_initialize
+ *
+ * Description:
+ *   Initialize a non-reentrant spinlock object to its initial,
+ *   unlocked state.
+ *
+ * Input Parameters:
+ *   lock  - A reference to the spinlock object to be initialized.
+ *   state - Initial state of the spinlock {SP_LOCKED or SP_UNLOCKED)
+ *
+ * Returned Value:
+ *   None.
+ *
+ ****************************************************************************/
+
+/* void spin_initialize(FAR spinlock_t *lock, spinlock_t state); */
```

Comment:
Please add new line after this comment line

Reply:
Done.